### ISSUE TITLE
Token Grabber Proof

### ISSUE THREAD

**Issue by codeuk** [mhmdnijem55/IMAGE-TOKEN-GRABBER-#2]
All of these 'Image Token Grabbers' are all grabbers themselves and the people behind them use spam accounts with cringe drill music on youtube to spread it, they're all harmless and the loggers aren't even coded by them.

I hope no one fell for this as it's really easy to tell it's fake.

**Comment by HYME60** [comment]
where can we find the real like this "Image token grabber" ? To much kid lol

**Comment by Rescore0** [comment]
I suggest just coding it urself as u cant trust them

**Comment by codeuk** [comment]
There's no actual 'Image Token Grabbers' as that's not how Discord works, any way to download scripts to the victims cache through images have been patched by Discord, and they didn't even grab the token.

These types of programs are only possible by the victim actively running a program / executing a script somehow. 

TL;DR Stop trying to look for 'Image Token Grabbers' as they don't exist and never will, they're all viruses
